pytorch_lightning/callbacks/quantization.py

```diff
@@ -219,3 +264,28 @@ def on_fit_end(self, trainer, pl_module):
             pl_module.forward = wrap_quantize_forward_context(model=pl_module, func=self.__module_forward)
         else:
             pl_module.forward = self.__module_forward
+
+    def on_validation_start(self, trainer, pl_module):
+        self._maybe_disable_observer(pl_module, 'validate' in self._disable_observers and not trainer.sanity_checking)
```

Comment:
```suggestion
        self._maybe_disable_observer(pl_module, 'validate' in self._disable_observers and trainer.sanity_checking)
```

Should we disable oberserver on sanity checking.

Reply:
In PyTorch 1.9, `FakeQuantize` with  `HistogramObserver` or `MovingAveragePerChannelMinMaxObserver` needs at least one batch to update the observer, or the quantization range computed by the observer would be incompatible with the `FakeQuantize`'s inputs.

If we disable observer during the sanity check, users may get `RuntimeError: dimensions of scale and zero-point are not consistent with input tensor`.